lib/cc/analyzer/engines_runner.rb

```diff
@@ -72,12 +75,16 @@ def metadata(engine_name)
         @registry[engine_name]
       end
 
-      def exclude_paths
-        PathPatterns.new(@config.exclude_paths || []).expanded + gitignore_paths
+      def include_paths
+        if paths && paths.any?
+          paths
+        else
+          IncludePathsBuilder.new(@config.exclude_paths || []).build
+        end
       end
 
-      def include_paths
-        IncludePathsBuilder.new(@config.exclude_paths || []).build
+      def exclude_paths
```

Comment:
Put `include_paths` at higher level than `exclude_paths` method because the former uses the latter, but maybe noisy change here.